addons/account/migrations/13.0.1.1/post-migration.py

```diff
@@ -0,0 +1,793 @@
+        )
+        openupgrade.logged_query(
+            env.cr, """
+            INSERT INTO account_tax_repartition_line_template (
+                account_id, factor_percent, invoice_tax_id,
+                refund_tax_id, repartition_type,
+                create_uid, create_date, write_uid, write_date)
+            SELECT NULL, 100, NULL, id, 'base',
+                create_uid, create_date, write_uid, write_date
+            FROM account_tax_template
+            WHERE id IN %s""", (tuple(tax_ids), ),
+        )
+        openupgrade.logged_query(
+            env.cr, """
+            INSERT INTO account_tax_repartition_line_template (
+                account_id, factor_percent, invoice_tax_id,
+                refund_tax_id, repartition_type,
+                create_uid, create_date, write_uid, write_date)
+            SELECT CASE WHEN tax_exigibility = 'on_payment'
+                THEN cash_basis_account_id ELSE refund_account_id END,
+                100, NULL, id, 'tax',
+                create_uid, create_date, write_uid, write_date
+            FROM account_tax_template
+            WHERE id IN %s""", (tuple(tax_ids), ),
+        )
+    openupgrade.logged_query(
+        env.cr, """
+        UPDATE account_tax_template att
+        SET cash_basis_transition_account_id = att.account_id
+        WHERE att.tax_exigibility = 'on_payment'
+            AND att.cash_basis_transition_account_id IS NULL"""
+    )
+
+
+def move_tags_from_taxes_to_repartition_lines(env):
+    openupgrade.logged_query(
+        env.cr, """
+        INSERT INTO account_account_tag_account_tax_repartition_line_rel (
+            account_tax_repartition_line_id, account_account_tag_id)
+        SELECT atrl.id, atat.account_account_tag_id
+        FROM account_tax_account_tag atat
+        JOIN account_tax_repartition_line atrl ON
+            (atat.account_tax_id = atrl.invoice_tax_id OR
+             atat.account_tax_id = atrl.refund_tax_id)"""
+    )
+    openupgrade.logged_query(
+        env.cr, """
+        INSERT INTO account_tax_repartition_financial_tags (
+            account_tax_repartition_line_template_id, account_account_tag_id)
+        SELECT atrlt.id, aatattr.account_account_tag_id
+        FROM account_account_tag_account_tax_template_rel aatattr
+        JOIN account_tax_repartition_line_template atrlt ON
+            (aatattr.account_tax_template_id = atrlt.invoice_tax_id OR
+             aatattr.account_tax_template_id = atrlt.refund_tax_id)"""
+    )
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    fill_account_reconcile_model_second_analytic_tag_rel_table(env)
+    type_change_account_reconcile_model_taxes(env)
+    map_res_company_fiscalyear_last_month(env)
+    fill_account_fiscal_position_company_id(env)
+    fill_account_account_type_internal_group(env)
+    map_account_journal_post_at_bank_rec(env)
+    fill_account_journal_restrict_mode_hash_table(env)
+    map_account_payment_term_line_option_after_invoice_month(env)
```

Comment:
Since odoo/odoo#57537 it was revert

Reply:
Yeah. I will update it when https://github.com/OCA/OpenUpgrade/pull/2409 gets merged.

Reply:
OK, Thanks.